### DISCUSSION TITLE
Side bar, command bar, and keyboard shortcuts aka hotkeys

### DISCUSSION THREAD

**Discussion by ghost** [squidowl/halloy#585] [General]
I love this client and the simplicity of it. But I hid my side bar and it took me a good 15 minutes of looking around to find the Keyboard Shortcuts section in the docs. I kept searching for hotkeys as I know them to be called, but they are not called hotkeys in the documentation.

I just wanted to start a discussion that coming in as a new user I was a little lost when I first accidentally hid my sidebar and couldn't get it back. 

A suggestion:  the UI could give a hint about the basic keyboard short cuts for returning the side bar, or getting back to the command bar, for the first time user.

**Comment by casperstorm** [comment]
You are not the first to mention it! In general the whole idea about hiding sidebar is a bit half baked right now as you mention.

**Reply by casperstorm** [reply]
See: https://github.com/squidowl/halloy/issues/524.

**Comment by ghost** [comment]
OK, it sounds like you guys are already on it. I was definitely hovering on the far-left area hoping it would pop back into view, as that is probably the most common interface pattern.

What about the documentation? If the word hotkey had been on this page https://halloy.squidowl.org/configuration/keyboard.html I think my first google search would have taken me straight to it.

**Reply by casperstorm** [reply]
Let me look at that later today 👍🏻
